### DISCUSSION TITLE
Is there a way to print a Text inside the Menu and Submenu?

### DISCUSSION THREAD

**Discussion by Makima-sama** [forntoh/LcdMenu#131] [Q&A]
![Untitled](https://github.com/forntoh/LcdMenu/assets/143432156/259ac266-a6d4-43fd-bef0-0514c5f46e0b)
Using the Submenu example provided with this Library. I just want to ask if there is a way to print a text where the arrow will not appear on the First row or on that text and instead will appear on the second row where the choices are.

// Define the main menu
MAIN_MENU(
    ITEM_BASIC("  Choose a Color "), //**<-- Not counted as a choice** the arrow will not appear here
    ITEM_BASIC("a. Black"),
    ITEM_SUBMENU("b. Red", settingsMenu),
    ITEM_BASIC("c. Orrange"),
    ITEM_BASIC("d. White")
);
/**
 * Create submenu and precise its parent
 */
SUB_MENU(settingsMenu, mainMenu,
    ITEM_BASIC("Backlight"), //**<-- Not counted as a choice** the arrow will not appear here
    ITEM_BASIC("Contrast")
);

LcdMenu menu(LCD_ROWS, LCD_COLS);

So Basically on the LCD I want it to look like this:
         Choose a Color
-> a. Black   
     b. Red          
     c. Orrange    
     d. White          

Instead of this:
->     Choose a Color
     a. Black
     b. Red
     c. Orrange
     d. White

Also does the method can also work in Submenu?

**Comment by forntoh** [comment]
That's not possible at the moment but it looks like something I could add to the library, although I have no ETA.
